src/rulesets/Base/array.jl

```diff
@@ -278,3 +278,128 @@ function rrule(::typeof(fill), value::Any, dims::Int...)
     end
     return fill(value, dims), fill_pullback
 end
+
+#####
+##### `findmax`, `maximum`, etc.
+#####
+
+for findm in (:findmin, :findmax)
+    findm_pullback = Symbol(findm, :_pullback)
+
+    @eval function frule((_, xdot), ::typeof($findm), x; dims=:)
+        y, ind = $findm(x; dims=dims)
+        return (y, ind), Tangent{typeof((y, ind))}(xdot[ind], NoTangent())
+    end
+
+    @eval function rrule(::typeof($findm), x::AbstractArray{<:Number}; dims=:)
+        y, ind = $findm(x; dims=dims)
+        project = ProjectTo(x)
+        function $findm_pullback((dy, _))
```

Comment:
Maybe? I am a bit confused about Tangent. I was trying things out with Zygote and they appear to work, but perhaps this would still work if the signature was  `findm_pullback(::Tangent)`.

Reply:
Ah, never mind, I thought the destructuring places a constraint. It's fine this way

Reply:
Added a comment & checked

Reply:
The method below `$findm_pullback(::Tuple{AbstractZero, Any})` will however not accept a Tangent. 

Should it be `Tangent{<:Any, <: Tuple{AbstractZero, Any}}`?

Reply:
(Now changed to a branch, seems simplest.)

Reply:
>The method below $findm_pullback(::Tuple{AbstractZero, Any}) will however not accept a Tangent.

this is fixed now, right?

Reply:
Yes, I killed that method, and just check the type after destructuring.